PathWeaver/output/Bean.wpilib.json

```diff
@@ -0,0 +1,1847 @@
+[
```

Comment:
Do we want to check these into VC? I think probably just so if we transfer to a new computer we don't have to re-generate them

Reply:
VC as in main branch? I cleaned up a lot of the unused paths, however some still may be useful for testing.